js/timepicker.js

```diff
@@ -485,6 +495,65 @@
       }
     }
 
+    _inputFromTextField() {
+      var isHours = this.currentView === 'hours';
+      var unit = Math.PI / (isHours ? 6 : 30);
+      var radius, radian;
+
+      if (this.options.twelveHour) {
+        radius = this.options.outerRadius;
+      }
+
+      if (isHours) {
+        var value = this['inputHours'].value;
+
+        if (!(value > 0 && value < 13)) {
+          this['inputHours'].value = new Date().getHours();
+        } else {
+          radian = value * unit;
+
```

Comment:
As there is a  
`if (not something) {doThis} else {doThat}`,

wouldn't it be easier to just do  
`if (something) {doThat} else {doThis}` ?

Reply:
Also agreed. Will be addressed.